### PR TITLE
fixed typo for page "Basics of communication"

### PR DESCRIPTION
**Pull request by chaensel-agco**

### CODE REVIEW

File: docs/communication.adoc [hunk 1]
```diff
@@ -71,12 +71,12 @@ The protocol layer describes the basic communicaton layer. All agrirouter commun
...
 === REST
 
-REST is a wellknown principle for online apis. For more information on REST, please see following recources:
+REST is a wellknown principle for online APIs. For more information on REST, please see following resources:
 
 * https://en.wikipedia.org/wiki/Representational_state_transfer
 * https://code.tutsplus.com/tutorials/a-beginners-guide-to-http-and-rest--net-16340
 
-REST uses HTTP requests, that result in an HTTP Response.
+REST uses HTTP requests, that results in a HTTP Response.
```
Comment (oliverrahner) [inline docs/communication.adoc:79]: In this case, "an" is correct

File: docs/communication.adoc [hunk 2]
```diff
@@ -114,15 +114,15 @@ MQTT is a subscription based protocol. Therefore, the client can be informed by
 
 * https://mqtt.org/
 
-The agrirouter provides an MQTT Broker serverside, so, an app instance has to connect to this server with its client.
+The agrirouter provides a MQTT Broker serverside, so, an app instance has to connect to this server with its client.
```
Comment (oliverrahner) [inline docs/communication.adoc:117]: In this case, "an" is correct

File: docs/communication.adoc [hunk 3]
```diff
@@ -114,15 +114,15 @@ MQTT is a subscription based protocol. Therefore, the client can be informed by
...
 
-agrirouter provides one MQTT Server per Endpoint, so there is no danger or chance to subscribe for messages of another endpoint
+The agrirouter provides one MQTT Server per Endpoint, so there is no danger or chance to subscribe for messages of another endpoint.
 
 ==== General setup
 
-An MQTT Connection requires 2 types of Certificates. On the one hand, there is a MQTT Server Certificate, for which you can find the keys link:./keys.adoc#MQTT-communication[here].
+A MQTT connection requires 2 types of certificates. On the one hand, there is a MQTT Server Certificate, for which you can find the keys link:./keys.adoc#MQTT-communication[here].
```
Comment (oliverrahner) [inline docs/communication.adoc:123]: In this case, "an" is correct

File: docs/communication.adoc [hunk 4]
```diff
@@ -114,15 +114,15 @@ MQTT is a subscription based protocol. Therefore, the client can be informed by
...
 
-This is required to setup an MQTT Connection over SSL.
+This is required to setup a MQTT Connection over SSL.
```
Comment (oliverrahner) [inline docs/communication.adoc:125]: In this case, "an" is correct

File: docs/communication.adoc [hunk 5]
```diff
@@ -161,7 +161,7 @@ Those messages still have to be requested through the inbox but are automaticall
 
 
 ==== Router Devices
-When using MQTT for a telemetry platform or a farming software with multiple accounts, multiple MQTT connections to different brokers would be required. To avoid this, agrirouter introduced a **router devices** which allows to use just one connection for multiple, virtual endpoints.
+When using MQTT for a telemetry platform or a farming software with multiple accounts, multiple MQTT connections to different brokers would be required. To avoid this, agrirouter introduced **router devices** which allows to use just one connection for multiple, virtual endpoints.
```
Comment (oliverrahner) [inline docs/communication.adoc:164]: allows --> allow

**Review by oliverrahner — COMMENTED** [review]
Thanks for your contribution!
Can you please correct the commented parts? 
That would be greatly appreciated!

File: docs/communication.adoc (hunk 5 above)
Comment (chaensel-agco) [reply docs/communication.adoc]: done

File: docs/communication.adoc (hunk 4 above)
Comment (chaensel-agco) [reply docs/communication.adoc]: done

File: docs/communication.adoc (hunk 3 above)
Comment (chaensel-agco) [reply docs/communication.adoc]: done

File: docs/communication.adoc (hunk 2 above)
Comment (chaensel-agco) [reply docs/communication.adoc]: done

File: docs/communication.adoc (hunk 1 above)
Comment (chaensel-agco) [reply docs/communication.adoc]: done

**Review by oliverrahner — APPROVED** [review]
Great! Thanks for your help.

**Pull request merged**